server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -198,6 +198,18 @@ public class PostgresWireProtocol {
     public static int SERVER_VERSION_NUM = 140000;
     public static String PG_SERVER_VERSION = "14.0";
 
+    @VisibleForTesting
+    static final String[][] PARAMETER_STATUS = new String[][] {
+        {"crate_version", Version.CURRENT.externalNumber()},
+        {"server_version", PG_SERVER_VERSION},
+        {"server_encoding", "UTF8"},
+        {"client_encoding", "UTF8"},
+        {"datestyle", "sessionSettings.dateStyle()"},
```

Comment:
Please check again the original code, `sessionSettings.dateStyle()` is a dynamic value we get from the session settings, so this `PARAMETER_STATUS` cannot be static.

It would be great to have a test to check that we send the correct `datestyle` if it's changed in a session.

Reply:
Thank you, reverted most of the change.


> It would be great to have a test to check that we send the correct datestyle if it's changed in a session.

I have done a quick manual testing and we do not communicate back any `datestyle` changes back to the client, opened an [issue](https://github.com/crate/crate/issues/15577) for this.